### PR TITLE
paulmoloneyr3/DOC-4832/410reqs

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
The following PRs updated [host-prereq.md](http://host-prereq.md/) for 4.9 and 4.8:

[ENT-8792 - Fixed mentions of Postgres 13.3, should be 13.8 by paulmoloneyr3 · Pull Request #730 · corda/corda-docs-portal](https://github.com/corda/corda-docs-portal/pull/730) 

[DOC-4294: 4.8: Platform support matrix and Host prerequisites and database requirements out of syncDONE](https://r3-cev.atlassian.net/browse/DOC-4294)  

However when 4.10 was merged into main, these changes weren’t included. Will update 4.10.

To summarise:

PostgreSQL changed from 13.3 to 13.8

Red Hat Enterprise Linux - 8.x added

Ubuntu Linux - 18.04 and 20.04 added

### CODE REVIEW

**Review by yarivus — COMMENTED** [review]
thx @paulmoloneyr3
